.github/CODEOWNERS

```diff
@@ -20,3 +20,11 @@ Jenkinsfile @daos-stack/build-and-release-watchers
 # any PR that touches client API or high level client code
 src/client @daos-stack/client-api-watchers
 src/include/daos_*.* @daos-stack/client-api-watchers
+
+# doc-watchers: files affecting documentation (docs, doxygen, etc.)
+mkdocs.yml @daos-stack/doc-watchers
+Doxyfile @daos-stack/doc-watchers
+docs/ @daos-stack/doc-watchers
+src/include/ @daos-stack/doc-watchers
```

Comment:
do you really want to get everything here or more like just external API ?

Reply:
doxygen precesses src/include/*.h so we should watch the same ... fixing